src/pages/users/Checkout.js

```diff
@@ -0,0 +1,176 @@
+              <label htmlFor="" className="form-label">Name</label>
+              <input type="text"  className="form-control"  value={name}  onChange={(e) => setName(e.target.value)}  />
+            </div>
+            <div className="form-group mb-3 col-md-6">
+              <label htmlFor="" className="form-label">Email</label>
+              <input type="email" className="form-control" value={email}  onChange={(e) => setEmail(e.target.value)} />
+            </div>
+            <div className="form-group mb-3 col-md-6">
+              <label htmlFor="" className="form-label">Phone number</label>
+              <input type="text" className="form-control" value={phone} onChange={(e) => setPhone(e.target.value)} />
+            </div>
+            <div className="form-group mb-3 col-md-6">
+              <label htmlFor="" className="form-label">Address</label>
+              <input type="text" className="form-control" value={address} onChange={(e) => setAddress(e.target.value)} />
+            </div>
+          </div>
+        </div>
+        <input type="hidden" name="user_id" value={2} />
+        <h4 className="checkout-heading text-center mt-4">Order information</h4>
+        <div className="form-container">
+          <div className="form-row mt-3">
+            <div className="form-group col-md-3">
+              <label htmlFor="product">Product name</label>
+            </div>
+            <div className="form-group col-md-3">
+              <label htmlFor="quantity">Quantity</label>
+            </div>
+            <div className="form-group col-md-3">
+              <label htmlFor="unit-price">Unit price</label>
+            </div>
+            <div className="form-group col-md-3">
+              <label htmlFor="total-price">Total price</label>
+            </div>
+            {carts.map((item, index) => (
+              <>
+                <div className="form-group col-md-3" key={index}>
+                  <label htmlFor="product">{item.product_name}</label>
+                </div>
+                <div className="form-group col-md-3" key={index}>
+                  <label htmlFor="quantity">X{item.quantity}</label>
+                </div>
+                <div className="form-group col-md-3" key={index}>
+                  <label htmlFor="unit-price">X{item.unit_price}</label>
+                </div>
+                <div className="form-group col-md-3" key={index}>
+                  <label htmlFor="total-price">{item.total_price}</label>
+                </div>
+              </>
+            ))}
+          </div>
+        </div>
+        <div className="form-container">
+          <div className="form-group col-md-12 text-right">
+            <hr
+              style={{
+                borderWidth: "2px",
+                borderColor: "#C4C4C4",
+                marginTop: "10px"
+              }}
+            />
+            <div
+              className="form-inline"
+              style={{
```

Comment:
Don't use style inline. Create css class for this one

Reply:
I have just resolved